src/pages/transaction-signing/components/event-card.tsx

```diff
@@ -1,34 +1,35 @@
 import React from 'react';
-import { Box, color, IconButton, Stack, Text } from '@stacks/ui';
-import { SpaceBetween } from '@components/space-between';
-import { FiMoreHorizontal as IconDots } from 'react-icons/fi';
-import { Caption } from '@components/typography';
-import { AssetItem } from '@pages/transaction-signing/components/asset-item';
+import {Box, color, IconButton, Stack, Text} from '@stacks/ui';
+import {SpaceBetween} from '@components/space-between';
+import {FiMoreHorizontal as IconDots} from 'react-icons/fi';
+import {Caption} from '@components/typography';
+import {AssetItem} from '@pages/transaction-signing/components/asset-item';
+import {SendFormSelectors} from "@tests/page-objects/send-form.selectors";
 
 export const TransactionEventCard: React.FC<any> = ({
-  title,
-  left,
-  right,
-  amount,
-  ticker,
-  icon,
-  message,
-  actions,
-  isLast,
-}) => {
+                                                      title,
+                                                      left,
+                                                      right,
+                                                      amount,
+                                                      ticker,
+                                                      icon,
+                                                      message,
+                                                      actions,
+                                                      isLast,
+                                                    }) => {
   return (
     <>
       <Stack spacing="base-loose" p="base-loose">
         <SpaceBetween position="relative">
           <Text fontWeight={500} fontSize={2}>
-            {title}
+            <span data-testid={SendFormSelectors.TransferMessage}>{title}</span>
```

Comment:
It isn't necessary to create a new `span` here. The `data-testid` selector can go on `Text`